### DISCUSSION TITLE
Bluescsi issue 50pin and db25

### DISCUSSION THREAD

**Discussion by herneise** [erichelgeson/BlueSCSI-v1#153] [Q&A]
Dear friends,
I bought 2 kits of bluescsi(50pin and db25). Soldered them. Additionally I bought smt32 board on aliexpress and made firmware update as here https://youtu.be/bRLAOts_MTM
Prepared image file as mentioned ( i am familiar with macintosh imaga files) but I couldn't run the both bluescsi adapters. Working already a week on them. 
Please help me to determine the error. Where should i check?
-sordering part seems ok since i have 2 different one. Checked twice. 
-image file also seems ok. Tried on basilisk. It is working. 
-is it possible because of smt32? I have only option to buy from aliexpress. Should i try another board?
Thanks.

**Comment by erichelgeson** [comment]
Please take a few minutes to go through the [troubleshooting guide ](https://github.com/erichelgeson/BlueSCSI/wiki/Troubleshooting)on the wiki and confirm you've completed the steps there, those usually fix 99% of issues.

Image files working in basilisk is not a sure thing - basilisk can boot both "drive" and "partition" images - bluescsi can only use real "drive" partitions. Try using a known good image or using Disk Jockey to check your images.

It is usually rare that a stm32 is bad, but can happen. If you bought kits from an authorized seller they will have tested the stm32 prior to shipping.
